pkg/vadmin/replication_start_vc.go

```diff
@@ -68,11 +68,11 @@ func (v *VClusterOps) genReplicateDBOptions(s *replicationstart.Parms, certs *HT
 	} else {
 		opts.TargetPassword = &s.TargetPassword
 	}
-	opts.TargetHosts = append(opts.TargetHosts, s.TargetIP)
+	opts.TargetHosts = append(opts.TargetHosts, s.TargetHostname)
 	opts.SourceTLSConfig = s.SourceTLSConfig
 	opts.IsEon = v.VDB.IsEON()
 
-	opts.IPv6 = net.IsIPv6(s.SourceIP)
+	opts.IPv6 = net.IsIPv6(s.SourceHostname)
```

Comment:
This only works with ips. Maybe we should just use the service clusterIP

Reply:
my issue is that if we use IPs then it could break tls. if the tls config being used for on the source side is validating hostnames it could validate the service name but not the ip address. in general we should prefer hostnames when tls might be involved. should i just remove this check?